### DISCUSSION TITLE
Homebrew Tap for Python app

### DISCUSSION THREAD

**Discussion by onpaws** [goreleaser/goreleaser#3240] [Q&A] [answered]
TIL about `goreleaser`, this seems to address a lot of common release needs, thanks!

Probably a long shot but thought I'd ask.
I have a Python CLI app that I'd like to publish as a Homebrew tap. Based on https://jondot.medium.com/shipping-rust-binaries-with-goreleaser-d5aa42a46be0 it seems like `goreleaser` might be able to help, but not 100% sure.

Are there any examples of Python + goreleaser I might consider? I briefly tried to track down `build-cross-release.sh` as mentioned on https://github.com/goreleaser/goreleaser/issues/962 but no luck.

Either way thanks :)

**Comment by caarlos0** [comment, marked as answer]
In theory, you could disable the build (`builds.disable = true`), add the `.py` file to the archive (although I'm not sure that would work), and craft the homebrew section.

There's no tutorial for that (that I'm aware of), but let me know if you manage to make it work :)

**Reply by onpaws** [reply]
Got it, thanks. If I manage to get somewhere I'll be happy to share 🤞
